sdk/file_sdk/UnitTests/xmp_ut/xmp_file_tests.cpp

```diff
@@ -3,8 +3,38 @@
 #include <QtTest>
 #include <QTStreamImpl.h>
 #include <XMPFileFormat/xmp_file_format.h>
+#include <CryptoAPI.h>
+#include <sstream>
+#include <fstream>
 
+using rmscrypto::api::IStream;
+using std::static_pointer_cast;
+using std::make_shared;
 using mip::file::XMPFileFormat;
+using mip::Tag;
+
+namespace {
+static bool VerifyTags(Tag tag1, Tag tag2) {
+  if (tag1.GetExtendedProperties().size() != tag2.GetExtendedProperties().size())
+    return false;
+
+  for (size_t i = 0; i < tag1.GetExtendedProperties().size(); i++) {
+    if (tag1.GetExtendedProperties()[i].key != tag2.GetExtendedProperties()[i].key ||
+        tag1.GetExtendedProperties()[i].value != tag2.GetExtendedProperties()[i].value ||
+        tag1.GetExtendedProperties()[i].vendor != tag2.GetExtendedProperties()[i].vendor) {
+      return false;

```

Comment:
you can use here == operator of ExtendedProperties.

Reply:
you can even do tag1.GetExtendedProperties() == tag2.GetExtendedProperties() without checking the size

---
In reply to: [136938770](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136938770) [](ancestors = 136938770)

Reply:
@Norfolc - what about order? does == ignores the order?

Reply:
@Alexander Riman - what about order? does == ignores the order?

---
In reply to: [136939146](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136939146) [](ancestors = 136939146,136938770)